app-misc/vcontrold/vcontrold-0.98.8.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
Use repoman to commit so it updates this copyright header automatically.. this is the old format.

Reply:
Also fixed. repoman also didn't complain anymore